EngineLayer/Proteomics/ProductTypeToTerminusType.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace EngineLayer
+{
+    static class ProductTypeToTerminusType
+    {
+        public static TerminusType IdentifyTerminusType(List<ProductType> lp)
+        {
+            if ((lp.Contains(ProductType.B) || lp.Contains(ProductType.BnoB1ions) || lp.Contains(ProductType.C)) && (lp.Contains(ProductType.Y) || lp.Contains(ProductType.Zdot)))
+            {
+                return TerminusType.None;
+            }
+            else if (lp.Contains(ProductType.Y) || lp.Contains(ProductType.Zdot))
```

Comment:
to make this future-proof, could you accomodate the a and x ions? Because right now this may return the wrong terminus type for those.